_layouts/wikipage.html

```diff
@@ -10,6 +10,6 @@
     {% include toc.html html=content id="toc" h_max=3 %}
   </div>
   <div class="col-xs col-sm-8 col-md-9 markdown-content fix-fouc">
-    {{ content | inject_anchors }}
```

Comment:
Good catch, this is likely a leftover from `jekyll-toc` which we removed very early as it is isn't compatible with GitHub Pages.